lib/floe/workflow/runner/podman.rb

```diff
@@ -55,6 +55,30 @@ def delete_secret(secret_guid)
           nil
         end
 
+        def parse_notice(notice)
+          id, status, exit_code = JSON.parse(notice).values_at("ID", "Status", "ContainerExitCode")
+
+          event   = podman_event_status_to_event(status)
+          running = event != :delete
+
+          container_state = {"Running" => running, "ExitCode" => exit_code.to_i}
```

Comment:
What defines the container state format?  That is, they seems different across all 3, and I can't tell if that's important or not (it feels like they should all return a consistent state format)

Reply:
It is intended to be "opaque" and only accessed by the runner so it doesn't have to be consistent (the pod inspect output is different between all three, external callers use the `running?` and `success?` interface methods.

Currently we're only checking Running and ExitCode so it could potentially be limited to just on all three that but it was originally the full `docker inspect` output